open_academy/models/session.py

```diff
@@ -20,3 +20,20 @@ def _compute_taken_seats(self):
             if record.number_of_seats > 0:
                 taken_seats = round((len(record.attendees_ids) * 100) / record.number_of_seats)
             record.taken_seats = taken_seats
+
+    @api.onchange('number_of_seats','attendees_ids')
+    def _onchange_invalid_values(self):
+        if self.number_of_seats < 0:
+            return {
+                'warning': {
+                'title': "Something bad happened.",
+                'message': "Number of seats must be greater than 0."
+                } 
+            }
+        if len(self.attendees_ids) > self.number_of_seats:
+            return {
+                'warning': {
+                'title': "Something bad happened.",
+                'message': "Number of attendees cannot be greater than the number of available seats."
+                } 
+            }
```

Comment:
Missing new line at the end of file.

Reply:
Old commit but on github it does not display the new line, inside the file itself it is

<img width="766" alt="Screen Shot 2022-08-26 at 1 30 03 p m" src="https://user-images.githubusercontent.com/47955967/186969190-5ab2502b-84ae-4a93-b933-8f46a8d6eaad.png">

Reply:
nice, just checked your message about this in the previous PR. Thanks for the info!